anonymizer/OpenDiffix.Service/Program.fs

```diff
@@ -35,17 +36,32 @@ let csvFormat value =
   | String string -> quoteString string
   | _ -> Value.toString value
 
-let csvFormatter result =
+let csvFormatter suppressedAnonCount result =
   let header =
     result.Columns
     |> List.map (fun column -> quoteString column.Name)
     |> String.join ","
 
+  let starBucketRow =
+    result.Columns
+    |> List.map (
+      function
+      | { Name = "count" } -> csvFormat suppressedAnonCount
+      | _ -> csvFormat (Value.String "*")
+    )
+    |> String.join ","
+
   let rows =
     result.Rows
     |> List.map (fun row -> row |> Array.map csvFormat |> String.join ",")
 
-  header :: rows |> String.join "\n"
+  match suppressedAnonCount with
+  // no suppression took place _OR_ the suppress bin was itself suppressed by the
```

Comment:
see https://github.com/diffix/desktop/pull/290/files#r764744040

Reply:
> no suppression took place

How can this happen?

Reply:
> SuppressedAnonLowCount: true | false | null

I am not sure what this means...

Reply:
>> no suppression took place
> How can this happen?

when all buckets pass the low count filter

> > SuppressedAnonLowCount: true | false | null
> 
> I am not sure what this means...

as you described elsewhere, better described by tri-state: "Star but suppressed, Star and visible, no star". `SuppressedAnonCount: null, SuppressedAnonLowCount: null` would represent `no star`, because there is "no star bucket".